### PR TITLE
uag: fix initializer

### PR DESCRIPTION
**Pull request by cspiel1**
part 1 of DnD feature: https://github.com/baresip/baresip/pull/2979

The initializer of uag is not correct.

### CODE REVIEW

**Comment by cspiel1** [comment]
Problems in  MaxOS build:
- `avcodec_close` is deprecated
- libavformat.dylib was built for newer macOS version (12.0) than being linked (10.12)

@sreimers should I make a PR for the first? For the second I have no idea.

**Comment by sreimers** [comment]
> * libavformat.dylib was built for newer macOS version (12.0) than being linked (10.12)

This can be ignored.

**Comment by cspiel1** [comment]
Here is the fix of the deprecated warning: https://github.com/baresip/baresip/pull/3002

File: src/uag.c [hunk 1]
```diff
@@ -12,23 +12,23 @@
...
+	NULL,         /**< exith             */
+	false,        /**< nodial            */
+	false,        /**< dnd               */
+	NULL,         /**< arg               */
+	NULL,         /**< eprm              */
 #ifdef USE_TLS
-	NULL,
-	NULL
+	NULL,         /**< tls               */
+	NULL          /**< wss_tls           */
```
Comment (cspiel1) [inline src/uag.c:31]: Should we replace this by `memset(&uag, 0, sizeof(uag))` in `ua_init()`?

Comment (sreimers) [reply src/uag.c:31]: You can replace this by:

`static struct uag uag = {.ual = LIST_INIT};`

All none specified members are initialized by 0 or similar zero type value.

**Pull request merged**